pkgs/applications/audio/giada/default.nix

```
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, libtool
+, fltk
+, libsndfile
+, libsamplerate
+, jansson
+, rtmidi
+, xorg
+, libjack2
+, alsaLib
+, libpulseaudio
+}:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "giada";
+  version = "0.15.4";
+
+  meta = with stdenv.lib;
+    { description = "Hardcore loop machine";
+      longDescription = ''
+        Giada is a free, minimal, hardcore audio tool for DJs, live performers
+        and electronic musicians. How does it work? Just pick up your channel,
+        fill it with samples or MIDI events and start the show by using this
+        tiny piece of software as a loop machine, drum machine, sequencer,
+        live sampler or yet as a plugin/effect host.
+      '';
+      homepage = https://www.giadamusic.com;
+      platforms = platforms.linux;
+      license = lib.licenses.gpl3;
```

```suggestion
      license = lib.licenses.gpl3Plus;
```

https://github.com/monocasual/giada/blob/251c67c3431b85ef3ac634e7348136cf8843b06d/src/main.cpp#L11-L14